phpstan.neon

```diff
@@ -5,11 +5,14 @@ parameters:
 	level: 1
 	paths:
 		- .
+	ignoreErrors:
+		- identifier: class.notFound
```

Comment:
Under MW 1.42+, DeferredUpdates is an alias so I think phpstan is choking on that. From what I read you have to manually define the file where the alias is at for it to understand.

Reply:
Yes, that could well be the case.